src/Logger.ts

```diff
@@ -59,7 +59,7 @@ export class Logger {
     }
   };
 
-  private buildLogEvent(level: string, params: any[]) {
+  private buildLogEvent(level: Lowercase<keyof typeof Level>, params: any[]) {
```

Comment:
The fix I had but that never got pushed it seems.
```suggestion
  private buildLogEvent(level: Event["level"], params: any[]) {
```